table/block_based/block_based_table_reader.cc

```diff
@@ -2430,6 +2461,10 @@ void BlockBasedTable::MultiGet(const ReadOptions& read_options,
       iiter_unique_ptr.reset(iiter);
     }
 
+    if (sst_file_range.begin()->get_context) {
+      ++(sst_file_range.begin()->get_context->get_context_stats_.num_sst_read);
```

Comment:
This shouldn't be incremented unconditionally, instead only when blocks have to be read. Line 2589 might be a better place to increment it.